### ISSUE TITLE
parallelize zar zq

### ISSUE THREAD

**Issue by alfred-landrum** [brimsec/zq#772]
Most of the zar operations can be parallelized to concurrently perform against the data or index files in the archive. Let's start with `zar zq`; this work can be the basis for parallelizing the rest of the operations.

**Comment by philrz** [comment]
@alfred-landrum: I know we'd done some parallelizing work lately, so with `zar` commit `56ed80d` I just ingested the wrccdc "year1" data set to make a 4.5 GB archive, then did:

```
$ zar zq -t 'count()'
#0:record[count:uint64]
0:[96217189;]
```

My per-second CPU chart showed `zar` utilizing about 8 of the 12 cores on my laptop.

![image](https://user-images.githubusercontent.com/5934157/94944739-89480e00-048e-11eb-82fe-5eb97103e265.png)

Can we say we've addressed this issue and can close it?

**Comment by alfred-landrum** [comment]
@philrz : yes, the final enabling of `zar zq` parallel operations landed in #1138 , closing.
